foreman_build_history.gemspec

```diff
@@ -7,15 +7,14 @@ Gem::Specification.new do |s|
   s.date        = Date.today
   s.authors     = ['Shim Shtein']
   s.email       = ['shteinshim@gmail.com']
-  s.homepage    = 'http://github.com'
-  s.summary     = 'Summary of ForemanBuildHistory.'
+  s.homepage    = 'http://github.com/theforeman/foreman_build_history'
```

Comment:
I am all for it, but I would like to add proper testing and rubocop before we do it.